KIPs/kip-34.md

```diff
@@ -476,9 +481,28 @@ None
 | createWithRoleBasedKey(address: String, roleBasedKeys: List&#60;String[]&#62;): RoleBasedKeyring | Creates a RoleBasedKeyring instance from an address and a 2D array of which each array element contains keys defined for each role. It throws an exception if the address string or the private key strings are invalid. |
 | decrypt(keystore: Object, password: String): Keyring | Decrypts a keystore v3 or v4 JSON and returns the decrypted keyring instance. It throws an exception if the decryption is failed. |
 
+#### IWallet
+
+`IWallet` is the interface of Wallet that manages data used when signing transactions. All Wallet classes must implement `IWallet`.
+Caver has a `KeyringContainer` instance that implements `IWallet`, and if it is a Wallet class that implements `IWallet` interface, it can be used in Caver instead of `KeyringContainer`.
```

Comment:
```suggestion
Caver has a class `KeyringContainer` that implements `IWallet`. If it is a class that implements `IWallet` interface, it can be used instead of `KeyringContainer` to sign a transaction.
```